docs/madr/decisions/011-multizone-envoy-admin.md

```diff
@@ -98,6 +105,25 @@ We can follow the same pattern as we did for CP <-> Envoy Admin.
 Generate internal CA, store it as a secret and generate server and client certs from it.
 We can verify SANs of certs because separate instance will have separate cert.
 
+#### Schema
+
+```grpc
+service InterCpService {
+  rpc Ping(PingRequest) returns (google.protobuf.Empty);
```

Comment:
I wouldn't use `google.protobuf.Empty` and just define a response already (even if it's empty) it just makes things easier to evolve.

Reply:
Good idea about the response 👍 

For the leader change. It will be spotted when followers retrieve a ConfigMap with instances. They need to do this anyways to pick an address.

Reply:
But what if the change happens in flight? Are folks rereading the configMap for each ping request?

Reply:
They do.
I guess we could introduce optimization to only read on the initial request and on leader change 👍